dockers/docker-lldp/lldpmgrd

```diff
@@ -148,6 +177,36 @@ class LldpManager(daemon_base.DaemonBase):
         for port_name in to_delete:
             self.pending_cmds.pop(port_name, None)
 
+    def lldp_check_mgmt_info_change(self):
+        (status, fvp) = self.device_table.get("localhost")
```

Comment:
do we support both v4 and v6 here ? How we are handling that ?

Reply:
Currently the lldpcli command allows configuring one of the IP address, it can be either IPv4 or IPv6. As part of making the changes for subscribe mode changes, will make the below change as well -
1) If both IPv4 and IPv6 are configured, then the latest IP address configured will be sent to lldpd process
2) When both are configured and one of them is deleted (say IPv6), then the other IP address (IPv4) will be sent to lldpd process